fcrepo6/rootfs/etc/confd/templates/setenv.sh.tmpl

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+export JAVA_OPTS="{{ getv "/java/opts" "" }}"
+export CATALINA_OPTS="{{ getv "/catalina/opts" (getenv "FCREPO_CATALINA_OPTS") }}"
+export CATALINA_OPTS="${CATALINA_OPTS} -Dfcrepo.home=/data/home"
+export CATALINA_OPTS="${CATALINA_OPTS} -Dfcrepo.velocity.runtime.log=/opt/tomcat/logs/velocity.log"
+export CATALINA_OPTS="${CATALINA_OPTS} -Dfcrepo.jms.baseUrl=http://{{ index (lookupIP (getenv "HOSTNAME")) 0 }}/fcrepo/rest"
+export CATALINA_OPTS="${CATALINA_OPTS} -Dfcrepo.external.content.allowed=/opt/tomcat/conf/allowed-external-content.txt"
+export CATALINA_OPTS="${CATALINA_OPTS} -Dfcrepo.autoversioning.enabled=false"
+export CATALINA_OPTS="${CATALINA_OPTS} -Dfcrepo.db.url=jdbc:{{ getv "/persistence/type" (getenv "FCREPO_PERSISTENCE_TYPE") }}://{{ getv "/db/host" (getenv "FCREPO_DB_HOST") }}:{{ getv "/db/port" (getenv "FCREPO_DB_PORT") }}/fcrepo"
```

Comment:
I assume there is no `repository.json` in fcrepo6 seems like the defaults here would cause problems? Like the default for `FCREPO_PERSISTENCE_TYPE` is file?